plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftClient.java

```diff
@@ -16,6 +16,7 @@
 import com.amazon.redshift.jdbc.RedshiftPreparedStatement;
```

Comment:
Please refrain from changing redshift now. We are missing CI for it. See https://github.com/trinodb/trino/pull/16142